open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-sync-connectors/ranger-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/securitysync/rangerconnector/admin/RangerConnector.java

```diff
@@ -23,11 +22,11 @@
 
         private static final Logger log = LoggerFactory.getLogger(RangerConnector.class);
 
-        public void initialize(SecuritySyncConfig securitySyncConfig, OMRSRepositoryConnector enterpriseConnector, OMRSAuditLog auditLog){
```

Comment:
SecuritySync can not have an enterprise connector because the repository services are not necesarily active in a governance deamon.

Reply:
Yes, I removed the Enterprise connector from this step, it is not needed.